File: frontend/src/constants/comment.ts
// 与 backend/internal/constants/comment.go 保持一致
export const CommentStatus = {
  PENDING: 'pending',
  APPROVED: 'approved',
  DELETED: 'deleted',
} as const;

export const CommentStatusLabels: Record<string, string> = {
  [CommentStatus.PENDING]: '待审核',
  [CommentStatus.APPROVED]: '已通过',
  [CommentStatus.DELETED]: '已删除',
};

File: backend/internal/constants/comment.go
package constants

// CommentStatus 评论状态枚举（README 枚举出现位置清单必列）。
const (
	CommentPending  = "pending"  // 待审核
	CommentApproved = "approved" // 已通过
	CommentDeleted  = "deleted"  // 已删除
)

// MaxReplyDepth 评论最多支持两层嵌套回复。
const MaxReplyDepth = 2

// CommentStatuses 全部评论状态。
var CommentStatuses = []string{CommentPending, CommentApproved, CommentDeleted}
